<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrmLogin</class>
 <widget class="QWidget" name="FrmLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Assets/Images/IMG-Login.png</normaloff>../../Assets/Images/IMG-Login.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>220</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>UserName:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>90</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>12</pointsize>
      <italic>true</italic>
     </font>
    </property>
    <property name="text">
     <string>PassWord:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textUserName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="textPassWord">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>261</width>
      <height>20</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnLogin">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Assets/Images/IMG-Login.png</normaloff>../../Assets/Images/IMG-Login.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnCancel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../Assets/Images/IMG-Cancel.png</normaloff>../../Assets/Images/IMG-Cancel.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
